framework/base/Security.php

```diff
@@ -642,66 +496,19 @@ public function validatePassword($password, $hash)
             throw new InvalidArgumentException('Hash is invalid.');
```

Comment:
is this check needed? no need to verify PHP internals, `password_verify` should do that for us.